### PR TITLE
Replace hidden label prefix with meta prefix.

### PR DESCRIPTION
**Pull request by fabxc**

### CODE REVIEW

File: model/labelname.go [hunk 1]
```diff
@@ -34,10 +34,8 @@ const (
 	// label names.
 	ReservedLabelPrefix = "__"
 
-	// HiddenLabelPrefix is a prefix which is legal in user-supplied label names
-	// but will not appear in the eventual metrics.
-	// Reserved labels may be excepted from that rule.
-	HiddenLabelPrefix = "_"
+	// MetaLabelPrefix is a prefix for labels that provide meta information.
```
Comment (juliusv) [inline model/labelname.go]: I would add something like "...that provide meta-information useful during target relabeling, but will not be attached to time series themselves."

Comment (fabxc) [inline model/labelname.go]: Certain yes to the final part. `Meta` is completely generic though and we might use it outside of SD in the future.

Comment (juliusv) [inline model/labelname.go]: :+1:

**Comment by fabxc** [comment]
Changed.

**Comment by juliusv** [comment]
:+1:

**Pull request merged**